client/src/pages/mocks/CreateMock.tsx

```diff
@@ -132,6 +132,9 @@ export const CreateMock: React.FC<CreateMockProps> = ({
         }}
         handleAddMockResponse={onAddResponse}
         handleDeleteMockResponse={onDeleteResponse}
+        // @ts-ignore
+        handleDuplicateMockResponse={undefined}/* TO PASS TS ERROR
```

Comment:
You can make `handleDuplicateMockResponse` as optional prop.
Than inside the component I saw that you have handled this scenario.

Reply:
What about this?